### PR TITLE
bug(EJ2-59406): Moved changes from old hotfix to latest.

### PR DESCRIPTION
**Pull request by indumathi-murali**
bug(BLAZ-59406): Moved changes from old hotfix to latest.

### CODE REVIEW

File: ej2-asp-core-mvc/linear-gauge/EJ2_ASP.MVC/getting-started.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The following steps are used to create ASP.NET MVC Application.
 
 ![Default Template](images/default-template.png)
 
-**Step 2:** Select **ASP.NET Web Application  (.NET Framework)** templates in Visual Studio 2019 and click **Next**.
+**Step 2:** Select **ASP.NET Web Application  (.NET Framework)** templates in visual studio 2019 and click **Next**.
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/linear-gauge/EJ2_ASP.MVC/getting-started.md:30]: Revert this.

File: ej2-asp-core-mvc/treemap/EJ2_ASP.NETCORE/getting-started.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ publishingplatform: ##Platform_Name##
 documentation: ug
 ---
 
-# Getting Started
+# Getting Started with ASP.NET Core TreeMap Control
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/treemap/EJ2_ASP.NETCORE/getting-started.md:11]: these changes are already available in hotfix.

File: ej2-asp-core-mvc/circular-gauge/EJ2_ASP.MVC/getting-started.md [hunk 3]
```diff
@@ -8,114 +8,125 @@ publishingplatform: ##Platform_Name##
 documentation: ug
 ---
 
-# Getting Started
+# Getting Started with ASP.NET MVC Circular Gauge Control
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/circular-gauge/EJ2_ASP.MVC/getting-started.md:11]: These changes are already in hotfix.

File: ej2-asp-core-mvc/linear-gauge/EJ2_ASP.NETCORE/getting-started.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 ---
 
 
-# Getting Started with ASP.NET Core LinearGauge
+# Getting Started with ASP.NET Core LinearGauge Control
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/linear-gauge/EJ2_ASP.NETCORE/getting-started.md:12]: already available in hotfix.

File: ej2-asp-core-mvc/maps/EJ2_ASP.NETCORE/getting-started.md [hunk 5]
```diff
@@ -170,40 +163,68 @@ The JSON object **"electionData.json"** is used as data source in the below code
 
 ```
 
-{% tabs %}
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/maps/EJ2_ASP.NETCORE/getting-started.md:173]: Removing this condition is wrong here.

**Comment by harivenkateshe** [comment]
Merge contents are irrelevant.

**Pull request closed without merging**